ArduCopter/heli.cpp

```diff
@@ -183,6 +182,20 @@ void Copter::heli_update_rotor_speed_targets()
                 motors.set_desired_rotor_speed(0);
             }
             break;
+        case ROTOR_CONTROL_MODE_CLOSED_LOOP_POWER_OUTPUT:
+            // pass setpoint through as desired rotor speed
+            if (rsc_control_deglitched > 0) {
+                motors.set_interlock(true);
+                if ((rpm_sensor.get_rpm(0) > 0) && ((heli_flags.using_governor_switch && heli_flags.governor_enable) || !heli_flags.using_governor_switch)){
+                    motors.set_rsc_governor_enabled(true, motors.get_gov_rpm_setpoint(), rpm_sensor.get_rpm(0));
```

Comment:
The reason I have the vehicle setting the target RPM, is because in the future, I want the vehicle to be able to vary the rotor RPM according to flight condition.  I haven't quite figured this out yet, but it's coming.  Think low RPM for hover, high RPM for high speed flight.

Also, this follows the pattern used by set_desired_rotor_speed above.

Now, that is not a bad idea to just send the RPM sensor right in.  I can probably do that.  I wanted to switch the sensor reading on/off, but I'm not sure there's really any point.